polkadot-protocol/protocol-components/transactions/transactions-formats.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Transactions Formats
+description: TODO
+---
+
+# Transactions Formats
+
+## Introduction
+
+This article describes the data structure of signed and unsigned transactions in Polkadot SDK-based chains. This is particularly useful for understanding how the transaction pool checks incoming transactions. Parachain builders will find this helpful in customizing how their transactions are formatted and writing client applications that must adhere to a chosen format.
+
+Extrinsics normally contain a signature, some data to describe whether the extrinsic has passed some validity checks and a reference to the pallet and call that it is intended for. This format provides a way for applications to ensure that the requirements for an extrinsic are met and correctly constructed.
+
+- Unchecked - signed transactions requiring some validation check before being accepted in the transaction pool. Any unchecked extrinsic contains the signature for the sent data and some extra data
```

Comment:
Might be a good idea to add some examples of unchecked, checked, and opaque types in the codebase